odoo/addons/base/ir/ir_model.py

```diff
@@ -1211,6 +1211,7 @@ def check(self, model, mode='read', raise_exception=True):
             else:
                 msg_tail = _("Please contact your system administrator if you think this is an error.") + "\n\n(" + _("Document model") + ": %s)"
                 msg_params = (model,)
+            msg_tail += ' - ({} {}, {}, {})'.format(_('Operation'), mode, _('User:'), self._uid)
```

Comment:
```suggestion
            msg_tail += ' - ({} {}, {}, {})'.format(_('Operation:'), mode, _('User:') self._uid)
```

To make align with other places too

![Screenshot from 2019-06-07 00-17-40](https://user-images.githubusercontent.com/16624719/59058179-afb96180-88b9-11e9-9a8e-c4ebdc017168.png)
Ps: Still it not good formatted IMO

Reply:
Do we have any reason to Display `uid` here ?
(As we have few only places to be called with  `sudo(user_id)`)

Reply:
Thanks, corrected. The PR is made to:
- avoid breaking the translations (hence the formatting is not consistent everywhere)
- provide all available information to our support service (some client may not mention the user facing the error)